engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -37,7 +36,6 @@
 import org.terasology.world.biomes.Biome;
 import org.terasology.world.biomes.BiomeManager;
 
-import java.util.List;
 import java.util.Locale;
 
 /**
```

Comment:
Please add class-level javadoc. In a short paragraph somebody looking at this class should immediately understand:
* what is the purpose of the DebugOverlay
* what fixed information it provides and in what format
* how the dynamic information is generated and handled, i.e. what methods manipulate the dynamic portion of the overlay

I agree this information should have been there in the first place, written by the original author of the class.